Modification style bloc vert

pierretusseau opened the pull request:
Il s'agit d'une modification du style du bloc utilisé sur le live, cette
modification a été demandé par S. Nocquard

pierretusseau committed:
Modification style bloc vert

Il s'agit d'une modification du style du bloc utilisé sur le live, cette
modification a été demandé par S. Nocquard

boogheta commented:
pas de souci pour merger, par contre j'ai l'impression que c'est imparfait : l'icone des groupes est déformée (étirée en largeur), il faut probablement aussi changer la height de l'image

boogheta commented:
Par ailleurs du coup la position de l'image du groupe bouge suivant la taille du texte, ce qui n'est peut-etre pas idéal pour l'incrustation fond vert, mais là c'est vous qui savez le mieux ;) cc @JeanMassiet

pierretusseau committed:
Correction hauteur de l'image du groupe

En effet la hauteur avait été oublié.
Pour ce qui est de la position du texte je pense laisser comme ça pour
l'instant, sans avoir de possibilité d'un aperçu direct c'est assez
difficile de me rendre compte du problème.

pierretusseau commented:
f0799ecff0a7a77d1a46ca0d83f64fb0613765bf

Voilà la correction, pour la position du texte je ne suis pas sûr de comprendre le problème par contre.

boogheta commented:
Merci! Pour le centrage du texte je laisse Jean voir si le résultat lui convient :)

pierretusseau commented:
Ok je me rends mieux compte du soucis avec la position du bloc, je vais voir avec Sylvain et Jean s'il y a besoin d'améliorer ça.
